api/v1beta1/manila_types.go

```diff
@@ -30,6 +30,14 @@ const (
 	SvcCleanupHash = "servicecleanup"
 	// DeploymentHash hash used to detect changes
 	DeploymentHash = "deployment"
+	// MessagingBus - TransportURLSecret messagingBus
+	MessagingBus = "messagingBus"
+	// NotificationBus - TransportURLSecret notifications
+	NotificationBus = "notificationBus"
+	// NotificationDriverNoop -
+	NotificationDriverNoop = "noop"
+	// NotificationDriverMessagingv2 -
+	NotificationDriverMessagingv2 = "messagingv2"
```

Comment:
its not super critical, but do we need those 2 driver const? could just handle all in the template

```
[oslo_messaging_notifications]
{{ if (index . "NotificationsURL") -}}
transport_url = {{ .NotificationsURL }}
driver = messagingv2
{{ else }}
driver = noop
{{ end -}}
```

Reply:
yes, that was basically my idea. seems thats how it was also done in the nova-op to have the condition in the template https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L139-L145

Reply:
ok I will switch to that approach in my next PS